config/element-api.php

```diff
@@ -336,7 +404,15 @@ function getListing($locale)
         'criteria' => $searchCriteria,
         'transformer' => function (Entry $entry) use ($locale, $pagePath) {
 
+            $status = 'live';
+            $revision = getDraftOrVersionOfEntry($entry);
```

Comment:
Any chance we could do this higher up like in the example so we are not fetching data twice?